### PR TITLE
Chrome user device characteristics report

### PR DESCRIPTION
**Pull request by jpmedley**
Fixes #SOME_ISSUE_NUMBER

I cannot find the tracking bug.

### CODE REVIEW

File: site/en/blog/device-characteristics/index.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+tags:
+  - device-characteristics
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/zIWz3bDI3mBRLki1VxMJ.jpeg'
+alt: >
+  An alternative text description of your hero or social image.
+---
+
+Developers are constantly wondering about what kind of audience they are
+developing for. Are they using high end devices with large RAM? What connection
+are they using (wifi, 2G...)? etc. The answers to such questions could carry
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:18]:
```suggestion
are they using (for example Wi-Fi, 2G) and so on. The answers to such questions could carry
```

File: site/en/blog/device-characteristics/index.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+much weight in many aspects of product development such as choosing the right
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:19]:
```suggestion
can affect product development decisions such as choosing the
```

File: site/en/blog/device-characteristics/index.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+technologies to render the content. This report provides timeline data to inform
+developers on these questions for a list of countries, like the US, Brazil,
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:21]:
```suggestion
developers on these questions for a list of countries, such as the US, Brazil,
```

File: site/en/blog/device-characteristics/index.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+Japan, etc. Three platforms are included when data is available: Android,
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:22]:
```suggestion
Japan, and others. Three platforms are included when data is available: Android,
```

File: site/en/blog/device-characteristics/index.md [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+Windows, and ChromeOS.
+
+Looking at the data, there might be some interesting things for developers to
+think about. For example, in Japan on the Android platform, the majority of
+users have five to eight cores, and the prevalence is quite a bit higher than
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:27]:
```suggestion
users have five to eight cores, and the prevalence is quite a bit higher than in
```

File: site/en/blog/device-characteristics/index.md [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+the US, and much higher than in NBU countries. Another example: if you look at
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:28]:
```suggestion
the US, and much higher than in NBU (next billion users) countries. Another example: if you look at
```

File: site/en/blog/device-characteristics/index.md [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+the smallest display size in device pixels, in some countries like the US or
+Brazil, there's a clear trend that smallest display sizes (like 400-500) are
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:30]: This seems like a weird way to refer to display size. 400px in which dimension, x, y, diagonal? Looking at this [chart of iPhone resolutions](https://www.ios-resolution.com/), some legitimately big phones such as 12 pro max are 428 x 926. (The chart says that's logical pixels, but I think it's mislabeled.) Also, consider adding a link to an explainer of device pixels vs logical pixels.

File: site/en/blog/device-characteristics/index.md [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+becoming more and more popular, while in some other countries like India, the
+mix looks pretty stable and there does not seem to be a trend. We hope
+publishing information like this periodically will help developers have a better
+understanding of the Chrome user device characteristics.
+
+This data comes from usage statistics sent to Google from Chrome installations.
+In Chrome, users can choose to send usage statistics and crash reports to Google
+to help improve Chrome's feature set and stability. Usage statistics and crash
+reports are enabled by default for Chrome consumer installs and can be disabled
+during installation or in Chrome's settings. Metrics collecting is pseudonymous,
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:40]:
```suggestion
during installation or in Chrome's settings. Metrics collection is pseudonymous,
```

**Comment by jpmedley** [comment]
@mihajlija The fixes weren't inline commits. Can you please look at the second paragraph again.

File: site/en/blog/device-characteristics/index.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+developers on these questions for a list of countries, such as the US, Brazil,
+Japan, and others. Three platforms are included when data is available: Android,
+Windows, and ChromeOS.
+
+Looking at the data, there might be some interesting things for developers to
+think about. For example, in Japan on the Android platform, the majority of
+users have five to eight cores, and the prevalence is quite a bit higher than in
+the US, and much higher than in next billion user (NBU) countries. Another
+example: if you look at RAM on Android, a very low percentage of Brazil devices
+have RAM larger than 5GB, which is much lower than in the US, and also lower
```
Comment (mihajlija) [inline site/en/blog/device-characteristics/index.md:30]:
```suggestion
have RAM larger than 5&nbsp;GB, which is much lower than in the US, and also lower
```

**Review by mihajlija — APPROVED** [review]

**Comment by jpmedley** [comment]
@MichaelSolati or @samthor This has a new author in it. I think it needs code approval as well.

**Comment by MichaelSolati** [comment]
When does this pull request need to land? The authors data has been completely restructured in main so Itd have to rebase and fixed. I wouldn't mind doing it but I wouldn't be able to for a couple hours

**Comment by jpmedley** [comment]
How long's the delay? A couple of hours? That would be fine. A week might be a problem.

**Comment by MichaelSolati** [comment]
@jpmedley don't worry, I'll work on it now.

**Review by MichaelSolati — APPROVED** [review]

**Comment by MichaelSolati** [comment]
@googlebot I consent.

**Comment by jpmedley** [comment]
Can someone please go into Percy and toggle the "Approved" switch so I can merge this?

File: site/en/blog/device-characteristics/index.md [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+to help improve Chrome's feature set and stability. Usage statistics and crash
+reports are enabled by default for Chrome consumer installs and can be disabled
+during installation or in Chrome's settings. Metrics collection is pseudonymous,
+and metrics are tagged by an opaque per-install identifier that is not joinable
+with other datasets.  If the metrics collection is re-enabled, the unique
+identifier will also be reset. We used the data collected to calculate the
+various statistics in this
+[report](https://docs.google.com/document/d/e/2PACX-1vTLB93N1X3QRs_0T869wIppeXYDGWeyDeVM5N8XSKVAx3cIl1RyNj3fM4gUn06cEimfUM4wubmC_laB/pub).
+
+Photo by [Halacious](https://unsplash.com/@halacious?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText) on [Upsplash](https://unsplash.com/s/photos/multiple-phones?utm_source=unsplash&utm_medium=referral&utm_content=creditCopyText)
```
Comment (samthor) [inline site/en/blog/device-characteristics/index.md:46]: nb: Upsplash should be Unsplash
Also, remove the `utm_source` and friends.

File: site/en/blog/device-characteristics/index.md [hunk 11]
```diff
@@ -0,0 +1,46 @@
...
+This data comes from usage statistics sent to Google from Chrome installations.
+In Chrome, users can choose to send usage statistics and crash reports to Google
+to help improve Chrome's feature set and stability. Usage statistics and crash
+reports are enabled by default for Chrome consumer installs and can be disabled
+during installation or in Chrome's settings. Metrics collection is pseudonymous,
+and metrics are tagged by an opaque per-install identifier that is not joinable
+with other datasets.  If the metrics collection is re-enabled, the unique
+identifier will also be reset. We used the data collected to calculate the
+various statistics in this
+[report](https://docs.google.com/document/d/e/2PACX-1vTLB93N1X3QRs_0T869wIppeXYDGWeyDeVM5N8XSKVAx3cIl1RyNj3fM4gUn06cEimfUM4wubmC_laB/pub).
```
Comment (samthor) [inline site/en/blog/device-characteristics/index.md:44]: I cannot access this document in Incognito or via my personal account.

File: site/en/blog/device-characteristics/index.md [hunk 12]
```diff
@@ -0,0 +1,46 @@
...
+description: >
+  Developers constantly wonder what kind of audience they are developing for. How much RAM do they have? What kind of Wi-fi are they on? A new report provides answers for a range of countries and platforms.
+date: 2021-09-22
+authors:
+  - jichen
+tags:
+  - device-characteristics
+hero: 'image/sQ51XsLqKMgSQMCZjIN0B7hlBO02/SJJ2a5QPJ2jSsNyGDbTh.jpeg'
+alt: >
+  An alternative text description of your hero or social image.
```
Comment (samthor) [inline site/en/blog/device-characteristics/index.md:13]: This needs to be filled out

**Review by samthor — APPROVED** [review]
Approved Percy changes, sorry. Noticed a few other minor problems.

**Pull request merged**